node/src/accountant/db_access_objects/sent_payable_dao.rs

```diff
@@ -540,23 +601,22 @@ mod tests {
             .timestamp(old_timestamp)
             .status(TxStatus::Failed)
             .build();
+        let tx4 = TxBuilder::default() // this should be picked for retry
```

Comment:
This comment will become misleading once we go off and implement the other table. The retry/not-retry decision will depend purely on what will be there, not this table. 

I just worry that you will forget to modify everything in the process of clean-up when you need to transfer some of the functions formerly planned for this table over to the other one. I trust your vow that you will go taking care of this in or after the next card is done, but there might be too many little things referring back to this state and you may miss some. It makes me a little nervous so please work with care.   

But I see now that this whole test will go away no matter what.